stable/elasticsearch/README.md

```diff
@@ -68,11 +68,12 @@ The following table lists the configurable parameters of the elasticsearch chart
 | `initImage.repository`               | Init container image name                                           | `busybox`                                           |
 | `initImage.tag`                      | Init container image tag                                            | `latest`                                            |
 | `initImage.pullPolicy`               | Init container pull policy                                          | `Always`                                            |
+| `cluster.enable`                     | Setup a cluster composed of several nodes (client, master, data). Otherwise deploys a single-node cluster. | `true`       |
```

Comment:
Typo spotted! :)

```suggestion
| `cluster.enabled`                     | Setup a cluster composed of several nodes (client, master, data). Otherwise deploys a single-node cluster. | `true`       |
```

Reply:
Fixed! Thanks!